src/resources/Microsoft.Authorization/policyDefinitions/Audit-AzureHybridBenefit.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "Audit-AzureHybridBenefit",
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "apiVersion": "2021-06-01",
+  "scope": null,
+  "properties": {
+    "policyType": "Custom",
+    "displayName": "Enable AHUB for eligible VMs",
+    "mode": "All",
+    "description": "Optimize cost by enabling Azure Hybrid Benefit. Leverage this Policy definition as a cost control to reveal Virtual Machines not using AHUB.",
+    "metadata": {
+      "version": "1.0.0",
+      "category": "Cost Optimization",
+      "source": "https://github.com/Azure/Enterprise-Scale/",
+      "alzCloudEnvironments": [
+        "AzureCloud",
+        "AzureChinaCloud",
+        "AzureUSGovernment"
+      ]
+    },
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
```

Comment:
You've defined an effect parameter, but you're not using it. Please use this parameter when applying the effect.